kivy/logger.py

```diff
@@ -115,8 +116,8 @@ def purge_logs(self, directory):
         if randint(0, 20) != 0:
             return
 
-        # Use config ?
-        maxfiles = 100
+        from kivy.config import Config
+        maxfiles = Config.getint('kivy', 'log_maxfiles') - 1
```

Comment:
This is a good point, we have to consider direct calls to `purge_logs()`. I am good with this and will remove the *-1* term. So the behaviour of *log_maxfiles* will be:

- `log_maxfiles <= 0`: disable purging, keep all logfiles.
- `log_maxfiles > 0`: purge logfiles and keep the most log_maxfiles recent logfiles. After that a new logfile will be added by the logger module so you will end up with `log_maxfiles+1` files.